middleware/server.js

```diff
@@ -0,0 +1,59 @@
+/*
+
+This file contains all endpoint available on the server
+and error handling
+
+*/
+
+const express = require("express");
+const { getTopics} = require("../controllers/general.controller");
+
+const server = express();
+server.use(express.json());
```

Comment:
Question: Is there any need for this middleware at this point? The only endpoint you have a GET request at this stage, so no parsing of the request body is needed at this stage. Try not to include things like this until you actually need them, as right now it's serving no purpose.